Dido's project/contact.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Contact Form</title>
+    <link rel="stylesheet" href="./StyleContact.css" />
+  </head>
+  <body class="background">
+    <main>
+      <div>
+        <header>
+          <h1 class="H1">Welcome to our contact form.</h1>
+          <p class="P1">
+            Please submit your contact details and a short detailed request
+            below.
+          </p>
+        </header>
+        <div>
+          <form class="Form">
+            <textarea placeholder="* Your Name Here"></textarea>
+            <textarea placeholder="* Your Email Here"></textarea>
+            <textarea placeholder="Your Phone Number Here"></textarea>
+            <textarea placeholder="* Your Inquiry Here"></textarea>
+            <button>Submit</button>
```

Comment:
Make sure you put a type of `type="submit"` on the button so when the user clicks on the button the form is submitted.